package.json

```diff
@@ -8,11 +8,11 @@
     "build-ts": "rimraf build && tsc",
     "watch-ts": "tsc -w",
     "serve": "npm run build-ts && ts-node build/server.js",
-    "migrate:up": "migrate up --migrations-dir cli/migrations",
-    "migrate:down": "migrate down --migrations-dir cli/migrations",
-    "migrate:create": "migrate create --migrations-dir cli/migrations --template-file cli/template.js",
     "compose:up": "docker-compose up",
     "compose:down": "docker-compose down",
+    "migration:make": "knex migrate:make --env local",
```

Comment:
I think i'd keep it as it is. I tried removing the `--env local` since making a migration shouldnt rely on which env you are on, but im getting a runtime error. 

Im going to add a comment tho, to make this one clear.

Reply:
I tried adding a comment on this json file, but adding comment in json is illegal, apparently :(